### PR TITLE
Add the Unikraft unikernel

### PR DESCRIPTION
**Pull request by vladandrew**
We add the Unikraft project to the Table of Contents, Existing projects and Comparing Solutions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -285,6 +290,7 @@ Out of the various existing projects, some standout due to their wide range of s
 | Rumprun | C, C++, Erlang, Go, Java, JavaScript, Node.js, Python, Ruby, Rust | Xen, KVM |  |
 | Unik | Go, Node.js, Java, C, C++, Python, OCaml | VirtualBox, ESXi, KVM, XEN, Amazon EC2, Google Cloud, OpenStack, PhotonController | Unikernel compiler toolbox with orchestration possible through Kubernetes and Cloud Foundry |
 | ToroKernel| FreePascal | VirtualBox, KVM, XEN, HyperV | Unikernel dedicated to run microservices |
+| Unikraft | C, C++, Rust, GO, Python etc. | KVM, XEN, Linuxu | Orchestration tool available |
```
Comment (banzo) [inline README.md]: typo here? 
"linuxu"

Comment (vladandrew) [reply README.md]: Right, this is actually linux userspace. Should I change it to just Linux?

Comment (banzo) [reply README.md]: yes please

**Comment by vladandrew** [comment]
Done, s/Linuxu/Linux/g

**Pull request merged**